_data/icons/ebtc.json

```diff
@@ -0,0 +1,8 @@
+[
+  {
+    "url":"ipfs://bafkreic4aq265oaf6yze7ba5okefqh6vnqudyrz6ovukvbnrlhet36itle",
+    "width":896,
```

Comment:
size seems to be 200x200
✓ ~/tmp ipfs get bafkreic4aq265oaf6yze7ba5okefqh6vnqudyrz6ovukvbnrlhet36itle
Saving file(s) to bafkreic4aq265oaf6yze7ba5okefqh6vnqudyrz6ovukvbnrlhet36itle
 33.39 KiB / 33.39 KiB [============================================] 100.00% 0s
✓ ~/tmp identify bafkreic4aq265oaf6yze7ba5okefqh6vnqudyrz6ovukvbnrlhet36itle
bafkreic4aq265oaf6yze7ba5okefqh6vnqudyrz6ovukvbnrlhet36itle PNG 200x200 200x200+0+0 8-bit sRGB 34195B 0.000u 0:00.000

Reply:
```suggestion
    "width":200,
```